src/main/java/org/hiero/sketch/spreadsheet/ApproxRank.java

```diff
@@ -0,0 +1,23 @@
+package org.hiero.sketch.spreadsheet;
+
+public class ApproxRank {
+    private final int lower;
```

Comment:
Instead of using get you can make these fields public - since they are final there's no harm.